src/cljs/main/broadfcui/common/icons.cljs

```diff
@@ -18,6 +18,7 @@
    :clone "fa-clone"
    :close "fa-close"
    :collapse "fa-minus-square-o"
+   :connection "fa-plug"
```

Comment:
Clever, but had to read the hover text to understand what it does

Just a thought, what about an "eye" <-> "eye with a slash through it" toggle?